### ISSUE TITLE
new Project Name?

### ISSUE THREAD

**Issue by surtic86** [surtic86/Project-Cars-DS-LiveApp#8]
I think we should get a new Project Name with that we can Publish our rewrite. What do you think about that?

**Comment by thmxv** [comment]
I do not have a lot of inspiration but to keep the analogy with liveracers, I would name it "PCARSLive". Or maybe "PCLWA" standing for "Project Cars Live Web App". The main issue with the current name is that it is too long and difficult to remember what is lower/upper case and when to add "-".

**Comment by surtic86** [comment]
Yes the Name right now is to long... we need something shorter and easier to keep in memory. I think "PCLWA" would be to abstract that you not really get on it that  it should be a Live Tracking App for Project Cars.
